<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoreSettingsForScan</class>
 <widget class="QDialog" name="MoreSettingsForScan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MoreSettingsForScan{
	background-image: url(:/Images/moreSettingForScan.png);
	background-repeat:no-repeat;
}</string>
  </property>
  <widget class="QComboBox" name="comboScanSize">
   <property name="geometry">
    <rect>
     <x>83</x>
     <y>325</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">
#comboScanSize {
	background: transparent;
}</string>
   </property>
   <item>
    <property name="text">
     <string>ResStr_A4_210_297mm_</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ResStr_A5_148_x_210mm_</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ResStr_B5_182_x_257mm_</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ResStr_Letter_8_5_x_11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ResStr_4_x_6_</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboDpi">
   <property name="geometry">
    <rect>
     <x>97</x>
     <y>160</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">
QComboBox{
	background: transparent;
}

QComboBox::down-arrow:on { /* shift the arrow when popup is open */
    top: 1px;
    left: 1px;
}</string>
   </property>
   <item>
    <property name="text">
     <string>100×100dpi</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>200×200dpi</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>300×300dpi</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>600×600dpi</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1200×1200dpi</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit_constrast">
   <property name="geometry">
    <rect>
     <x>353</x>
     <y>195</y>
     <width>50</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="statusTip">
    <string extracomment="value range in25-500"/>
   </property>
   <property name="styleSheet">
    <string notr="true">
#lineEdit_constrast{
	border:transparent;
}</string>
   </property>
   <property name="text">
    <string>100</string>
   </property>
   <property name="maxLength">
    <number>3</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>-4</x>
     <y>27</y>
     <width>530</width>
     <height>411</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
	border:transparent;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="btPthoto">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>94</x>
      <y>53</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
QRadioButton::indicator {
    width: 25px;
    height: 25px;
}

QRadioButton::indicator::unchecked {
	image: url(:/Images/Uncheck.png);
}


QRadioButton::indicator::checked {
	image: url(:/Images/Checked.png);
}
</string>
    </property>
    <property name="text">
     <string>ResStr_Photo_2</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="btPhotoText">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>53</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
QRadioButton::indicator {
    width: 25px;
    height: 25px;
}

QRadioButton::indicator::unchecked {
	image: url(:/Images/Uncheck.png);
}


QRadioButton::indicator::checked {
	image: url(:/Images/Checked.png);
}
</string>
    </property>
    <property name="text">
     <string>ResStr_Text_and_Graph</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="btText">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>53</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
QRadioButton::indicator {
    width: 25px;
    height: 25px;
}

QRadioButton::indicator::unchecked {
	image: url(:/Images/Uncheck.png);
}


QRadioButton::indicator::checked {
	image: url(:/Images/Checked.png);
}
</string>
    </property>
    <property name="text">
     <string>ResStr_Text</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider_contrast">
    <property name="geometry">
     <rect>
      <x>304</x>
      <y>148</y>
      <width>156</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QSlider::groove:horizontal {
	
	border-image: url(:/Images/silderLine.png);
   	border: 0px;
   	height: 2px;	
/*	background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #B1B1B1, stop:1 #c4c4c4);*/
	margin: 2px 0;
}

QSlider::handle:horizontal {
  	background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #b4b4b4, stop:1 #8f8f8f);
    border: 1px solid #5c5c5c; 
    width: 8px;
	margin: -7px 0;
	border-image: url(:/Images/silderBtn.png);
    border-radius: 4px;
}

</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_DocType">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>18</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ResStr_Document_Type</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="slider_brightness">
    <property name="geometry">
     <rect>
      <x>304</x>
      <y>267</y>
      <width>156</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QSlider::groove:horizontal {
	border-image: url(:/Images/silderLine.png);
   	border: 0px;
   	height: 2px;	
/*	background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #B1B1B1, stop:1 #c4c4c4);*/
	margin: 2px 0;
}

QSlider::handle:horizontal {
 
	border-image: url(:/Images/silderBtn.png);
 	background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #b4b4b4, stop:1 #8f8f8f);
    border: 1px solid #5c5c5c; 
    width: 8px;
	margin: -7px 0;
 
    border-radius: 4px; 
}

</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_DPI">
    <property name="geometry">
     <rect>
      <x>101</x>
      <y>105</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ResStr_DPI</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_ScanSize">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>270</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ResStr_Scan_Size</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Contrast">
    <property name="geometry">
     <rect>
      <x>319</x>
      <y>105</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ResStr_Contrast</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Brightness">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>223</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ResStr_Brightness</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btBgColor">
    <property name="geometry">
     <rect>
      <x>59</x>
      <y>206</y>
      <width>51</width>
      <height>38</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
#btBgColor{
	background-color: rgb(211, 211, 211);
	 border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btBgGrayscale">
    <property name="geometry">
     <rect>
      <x>128</x>
      <y>206</y>
      <width>51</width>
      <height>38</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
#btBgGrayscale{
	background-color: rgb(211, 211, 211);
	 border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btBgBW">
    <property name="geometry">
     <rect>
      <x>194</x>
      <y>206</y>
      <width>51</width>
      <height>38</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
#btBgBW{
	background-color: rgb(211, 211, 211);
	 border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btModeColor">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>209</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
#btModeColor {
	
	border-image: url(:/Images/Color.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btModeGreyscale">
    <property name="geometry">
     <rect>
      <x>138</x>
      <y>209</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
#btModeGreyscale {
	
	border-image: url(:/Images/Grayscale.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btModeBW">
    <property name="geometry">
     <rect>
      <x>204</x>
      <y>209</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
#btModeBW {
	border-image: url(:/Images/BlackWhite.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Brightness_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>177</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ResStr_Color_Mode</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btContrastAdd">
    <property name="geometry">
     <rect>
      <x>439</x>
      <y>170</y>
      <width>26</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
#btContrastAdd{
	border-image: url(:/Images/Btn_BackGround.png);
}


#btContrastAdd:hover{
	color: rgb(0, 170, 0);
}

#btContrastAdd:pressed{
	border-image: url(:/Images/Btn_ClickBackGround.png);
	color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btContrastReduce">
    <property name="geometry">
     <rect>
      <x>302</x>
      <y>170</y>
      <width>26</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
#btContrastReduce{
	border-image: url(:/Images/Btn_BackGround.png);
}


#btContrastReduce:hover{
	color: rgb(0, 170, 0);
}

#btContrastReduce:pressed{
	border-image: url(:/Images/Btn_ClickBackGround.png);
	color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>—</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btBrightnessAdd">
    <property name="geometry">
     <rect>
      <x>439</x>
      <y>290</y>
      <width>26</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
#btBrightnessAdd{
	border-image: url(:/Images/Btn_BackGround.png);
}


#btBrightnessAdd:hover{
	color: rgb(0, 170, 0);
}

#btBrightnessAdd:pressed{
	border-image: url(:/Images/Btn_ClickBackGround.png);
	color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btBrightnessReduce">
    <property name="geometry">
     <rect>
      <x>302</x>
      <y>290</y>
      <width>26</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
#btBrightnessReduce{
	border-image: url(:/Images/Btn_BackGround.png);
}


#btBrightnessReduce:hover{
	color: rgb(0, 170, 0);
}

#btBrightnessReduce:pressed{
	border-image: url(:/Images/Btn_ClickBackGround.png);
	color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>—</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_brightness">
    <property name="geometry">
     <rect>
      <x>357</x>
      <y>287</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="statusTip">
     <string extracomment="value range in25-500"/>
    </property>
    <property name="styleSheet">
     <string notr="true">
#lineEdit_brightness{
	border:transparent;
}</string>
    </property>
    <property name="text">
     <string>100</string>
    </property>
    <property name="maxLength">
     <number>3</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btOK">
    <property name="geometry">
     <rect>
      <x>282</x>
      <y>350</y>
      <width>130</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">
#btOK {
	border-image: url(:/Images/Btn_Green.png);
}

#btOK:pressed {
	border-image: url(:/Images/Btn_Green_Pressed.png);
	
	color: rgb(255, 255, 255);
}

#btOK:hover {
	color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>ResStr_OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btDefault">
    <property name="geometry">
     <rect>
      <x>117</x>
      <y>350</y>
      <width>130</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
#btDefault {
	border-image: url(:/Images/Btn_Gray.png);
}

#btDefault:hover {
	border-image: url(:/Images/Btn_Gray.png);
	color: rgb(0, 170, 0);
}

#btDefault:pressed {
	border-image: url(:/Images/Btn_Gray_Pressed.png);
	color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>ResStr_Default</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_tip">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>200</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(176, 255, 176);</string>
    </property>
    <property name="text">
     <string>ResStr_The_valid_range_is_0_100__please_confirm_and_enter_again_</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btContrastBar">
    <property name="geometry">
     <rect>
      <x>304</x>
      <y>135</y>
      <width>156</width>
      <height>11</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Images/Constrast.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btBrightnessBar">
    <property name="geometry">
     <rect>
      <x>304</x>
      <y>254</y>
      <width>156</width>
      <height>11</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Images/Brightness.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btClose">
   <property name="geometry">
    <rect>
     <x>506</x>
     <y>5</y>
     <width>13</width>
     <height>13</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
#btClose {
	border-image: url(:/Images/cancelPress.png);
}

#btClose:hover {
	border-image: url(:/Images/cancelMouseMove.png);
}

#btClose:pressed {
	border-image: url(:/Images/cancelIcon.png);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>0</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>ResStr_More_Settings</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
